FidXS_13TeV_2018_80X_preM19_newBinning_preApp/efficiencyFactors.py

```diff
@@ -675,7 +710,7 @@ def geteffs(channel, List, m4l_bins, m4l_low, m4l_high, obs_reco, obs_gen, obs_b
 if (opt.OBSNAME == "Phi1"):
     obs_reco = "abs(Phi1)"
     obs_gen = "abs(GENPhi1)"
-    
+'''    
```

Comment:
Hi @tjavaid 

Why you commented the lines 668 to 713?

Reply:
Hi @ram1123 , I just generalised this part a bit. Since now we have many additional variables which might make the script more messy. Added lines 656-668 can do the same job.

Reply:
hmmm this is good but I don't see the generalised information.

By generalisation I am expecting that you added a loop or something.

But I see that you just keep

```python
if (opt.OBSNAME == "rapidity4l"):
     obs_reco = "abs(rapidity4l)"
     obs_gen = "abs(GENrapidity4l)"
```

and commented all if condition for other observables.